docs/topics/templates.txt

```diff
@@ -441,24 +444,43 @@ adds defaults that differ from Jinja2's for a few options:
     Unless all of these conditions are met, passing a function to the template is
     simpler and more in line with the design of Jinja2.
 
-The default configuration is purposefully kept to a minimum. If a template is
-rendered with a request (e.g. when using :py:func:`~django.shortcuts.render`),
-the ``Jinja2`` backend adds the globals ``request``, ``csrf_input``, and
-``csrf_token`` to the context. Apart from that, this backend doesn't create a
-Django-flavored environment. It doesn't know about Django filters and tags.
-In order to use Django-specific APIs, you must configure them into the
-environment.
+If a template is rendered with a request (e.g. when using
+:py:func:`~django.shortcuts.render`), the ``Jinja2`` backend adds the globals
+``request``, ``csrf_input``, and ``csrf_token`` to the context. Apart from
+that, this backend does not create a Django-flavored environment. It doesn't
+know about Django filters and tags. In order to use Django-specific APIs, you
+must configure them into the environment.
+
+The default ``'django.template.backends.jinja2.default_environment'``
+environment adds some minimal helpers: it gives access to a ``url`` function,
```

Comment:
Not quite sure about "...helpers..." here.

Reply:
I'm definitely open to suggestions here. I wasn't sure what else to describe them as, since they're not exactly "template tags".